packages/@aws-cdk/core/test/test.staging.ts

```diff
@@ -0,0 +1,77 @@
+import * as cxapi from '@aws-cdk/cx-api';
```

Comment:
Why is this also not a move?

Reply:
Intentionally left the tests in the original module to ensure we maintain backwards compatibility.